src/System.Memory/tests/ReadOnlyMemory/CtorArray.cs

```diff
@@ -41,6 +41,7 @@ public static void CtorArrayLong()
             memory.Validate(91, -92, 93, 94, -95);
         }
 
+        [ActiveIssue(20884, TargetFrameworkMonikers.UapAot)]
```

Comment:
Why are you linking it to a System.Drawing issue? #20884

Reply:
Fixed. Thanks for catching it.